app/src/main/java/com/locoquest/app/dto/LocationDetails.kt

```diff
@@ -3,4 +3,5 @@ package com.locoquest.app.dto
 data class LocationDetails(
     val longitude: String,
     val latitude: String,
+    val name: String,
```

Comment:
This looks like an incomplete feature.
Is the name attribute used anywhere?  
If not in use, remove them.  Keep the codebase as small as possible to do the job you want to do.  Unused code is clutter, and bad documentation, which increases technical debt.
Only complete features, which meet the team's Definition of Done, should be merged to Main/Master.